deployment/network-operator/README.md

```diff
@@ -339,6 +339,8 @@ imagePullSecrets:
 | `ofedDriver.version` | string | `5.5-1.0.3.2` | Mellanox OFED driver version                                                                                                                                              |
 | `ofedDriver.imagePullSecrets` | list | `[]` | An optional list of references to secrets to use for pulling any of the Mellanox OFED driver image                                                                        |
 | `ofedDriver.env` | list | `[]` | An optional list of [environment variables](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#envvar-v1-core) passed to the Mellanox OFED driver image |
+| `ofedDriver.repoConfig.configMapName` | string | `` | Private mirror repository configuration configMap name |
```

Comment:
i feel like these two should be named the same : either `name` or `configMapName` WDYT ?

need to align thoroughout the PR (helm, CRD, code)

Reply:
I think this is to align with NVIDIA gpu-operator.

Reply:
these two parameters are very similar, and should be named the same for consistency.

we do not have a strict requirement to have the same api as gpu operator. where we can improve / provide a more consistent api , i think we should.

Reply:
@e0ne please also change here in the README

Reply:
done